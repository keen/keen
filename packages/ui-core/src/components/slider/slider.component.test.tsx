import React from 'react';
import { shallow } from 'enzyme';
import Slider from './slider.component';

describe('<Slider />', () => {
  const colors = ['white', 'blue'];

  test('should render Slider component', () => {
    const wrapper = shallow(<Slider colors={colors} />);
    expect(wrapper).toMatchSnapshot();
  });
});
