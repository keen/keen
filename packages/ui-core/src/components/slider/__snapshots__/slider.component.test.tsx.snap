// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Slider /> should render Slider component 1`] = `
<Fragment>
  <div
    style={
      Object {
        "background": "linear-gradient(90deg, white, blue",
        "borderRadius": 3,
        "height": 4,
        "position": "relative",
        "width": "100%",
      }
    }
  >
    <Control
      dragConstraints={
        Object {
          "bottom": 0,
          "left": 0,
          "right": 0,
          "top": 0,
        }
      }
      initialPos={0}
      isHorizontal={true}
      max={100}
      min={0}
      number={1}
      onDrag={[Function]}
      sliderSize={0}
      steps={0}
      tooltip={
        Object {
          "enabled": true,
          "position": "bottom",
        }
      }
      zIdx={true}
    />
    <Component
      layout="horizontal"
      ticks={
        Array [
          Object {
            "pos": 0,
            "val": 0,
          },
          Object {
            "pos": 0,
            "val": 25,
          },
          Object {
            "pos": 0,
            "val": 50,
          },
          Object {
            "pos": 0,
            "val": 75,
          },
          Object {
            "pos": 0,
            "val": 100,
          },
        ]
      }
    />
  </div>
</Fragment>
`;
